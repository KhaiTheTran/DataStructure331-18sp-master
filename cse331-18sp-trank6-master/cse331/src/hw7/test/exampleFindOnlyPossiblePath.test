# Find a path with Dijkstra's algorithm when there is only one possible path

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK
AddNode characters VENUS-II
AddEdge characters GORILLA-MAN HAWK 1
AddEdge characters HAWK VENUS-II 1.5

FindPath characters GORILLA-MAN VENUS-II
