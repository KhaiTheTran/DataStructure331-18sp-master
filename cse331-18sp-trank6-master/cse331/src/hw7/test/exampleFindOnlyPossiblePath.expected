# Find a path with Dijkstra's algorithm when there is only one possible path

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters
added node VENUS-II to characters
added edge 1.000 from GORILLA-MAN to HAWK in characters
added edge 1.500 from HAWK to VENUS-II in characters

path from GORILLA-MAN to VENUS-II:
GORILLA-MAN to HAWK with weight 1.000
HAWK to VENUS-II with weight 1.500
total cost: 2.500
